website/common/script/content/hatching-potions.js

```diff
@@ -290,9 +290,9 @@ each(wacky, (pot, key) => {
     notes: t('hatchingPotionNotes', {
       potText: pot.text,
     }),
-    _addlNotes: t('eventAvailability', {
+    _addlNotes: pot._seasont && pot._season !== '_PENDING_' ? ('eventAvailability', {
```

Comment:
should this be `pot._season` without an ending `t`?

Reply:
Yes you are right, I added the `t` :)